### PR TITLE
Fix `columns` prop type when empty (array)

### PR DESCRIPTION
**Pull request by distantnative**

### CODE REVIEW

**Review by afbora — DISMISSED** [review]
LGTM

File: panel/src/components/Layout/Collection.vue [hunk 1]
```diff
@@ -57,7 +57,7 @@
  */
 export default {
   props: {
-    columns: Object,
+    columns: [Object, Array],
```
Comment (afbora) [inline panel/src/components/Layout/Collection.vue]:
```suggestion
    columns: {
      type: [Object, Array],
      default() {
        return {};
      }
    },
```

File: panel/src/components/Layout/Items.vue [hunk 2]
```diff
@@ -44,7 +44,7 @@
 export default {
   inheritAttrs: false,
   props: {
-    columns: Object,
+    columns: [Object, Array],
```
Comment (afbora) [inline panel/src/components/Layout/Items.vue]:
```suggestion
    columns: {
      type: [Object, Array],
      default() {
        return {};
      }
    },
```

File: panel/src/components/Layout/Collection.vue (hunk 1 above)
Comment (afbora) [reply panel/src/components/Layout/Collection.vue]: I think it will be more stable that way.

**Review by bastianallgeier — APPROVED** [review]

**Pull request merged**